tools/interactive/interactivetool_metashark.xml

```diff
@@ -0,0 +1,55 @@
+<tool id="interactive_tool_metashark" tool_type="interactive" name="metashark" version="1.6">
```

Comment:
version number does not fit the container ? maybe as a macro token?

Reply:
You're right! Thank you!